### DISCUSSION TITLE
CKEditor 5 Plugins

### DISCUSSION THREAD

**Discussion by jkenney3** [requarks/wiki#5485] [Help / Questions] [answered]
I'm trying to add some additional buttons into the CKEditor integrated into wiki.js. Specifically the ckeditor5-html-embed. When I put the files in the node_modules folder with the other plug-ins nothing seems to happen. If I try to build for some reason, recursive symbolic links are created. 

So I have to rebuild the entire wiki codebase to add the plugin or can I just rebuild CKEditor?

I appreciate any help in advance!

**Comment by NGPixel** [comment, marked as answer]
You would need to rebuild the entire client-side codebase. There's no easy way to add CKEditor plugins at the moment.

**Reply by jkenney3** [reply]
That's what I suspected, thank you for the quick response!
